### PR TITLE
Train config

### PR DESCRIPTION
**Pull request by DianaDI**
Refactor;
Add most successful train config for SSDnet

### CODE REVIEW

File: TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config [hunk 1]
```diff
@@ -139,8 +139,8 @@ train_config: {
       learning_rate: {
         exponential_decay_learning_rate {
           initial_learning_rate: 0.01
```
Comment (hitanshsingla) [inline TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config:141]: I'd suggest a higher lr, around 0.0001

File: TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config [hunk 2]
```diff
@@ -139,8 +139,8 @@ train_config: {
...
-          decay_steps: 400
-          decay_factor: 0.95
+          decay_steps: 300
```
Comment (hitanshsingla) [inline TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config:142]: and this could be 500

File: TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config [hunk 3]
```diff
@@ -155,26 +155,6 @@ train_config: {
   # effectively bypasses the learning rate schedule (the learning rate will
   # never decay). Remove the below line to train indefinitely.
   num_steps: 15000
```
Comment (hitanshsingla) [inline TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config:157]: In my opinion, 15k are much higher, because it would take a lot of time.
Shall we make it 10k? We already have to do around 10+ variations

File: TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config (hunk 1 above)
Comment (DianaDI) [reply TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config:141]: 0.0001 is lower, not higher. It gives worse results in the beginning and in the end. I have tried. With higher/bigger LR we get faster convergence in the beginning, while we are far away from optimum

File: TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config (hunk 2 above)
Comment (DianaDI) [reply TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config:142]: I found 300 is better according to the loss behaviours.

Comment (DianaDI) [reply TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config:142]: even better than 400

**Review by hitanshsingla — APPROVED** [review]

File: TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config (hunk 3 above)
Comment (DianaDI) [reply TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config:157]: ok, 10k gave me same loss. maybe at least 12k, because of on loss plot 10k it seems like 10k not fully explores loss
